.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing to HTTP Methods
```

Comment:
Mind moving this into the root of the repository?

Reply:
Done!